source/install.md

```diff
@@ -0,0 +1,245 @@
+
+```shell-session
+$ mamba env create -n <name> -f <path to .yaml file>
+```
+
+Any time you want to use software installed in an environment you have created, you must activate it. An environment will stay active until you close the terminal window or shell session:
+
+```shell-session
+$ conda activate <name>
+$ jupyter notebook .
+```
+
+[virtual environments]: conda:user-guide/concepts/environments
+[local to your user]: install_mamba
+
+### Advanced Environment Management
+
+If you prefer, you can specify the path to a virtual environment's root directory as a prefix instead of giving an environment's name:
+
+```shell-session
+$ mamba create -c conda-forge -p <prefix> <package(s)>
+$ conda activate <prefix>
+```
+
+If you specify neither the name `-n` nor the prefix `-p`, the current active environment will be used.
+
+New software can be installed in an environment by listing packages:
+
+```shell-session
+$ mamba install -n <name> -c conda-forge <package(s)>
+```
+
+Or by adding the dependencies from an environment file:
+
+```shell-session
+$ mamba env update -n <name> -f <path to .yaml file>
+```
+
+Mamba/Conda cache the packages you install, so if you've already downloaded a particular version, you can install it in as many environments as you like without having to re-download it. In fact, Mamba/Conda try their hardest to unpack each version once, and then hardlink to it from each environment to keep the storage size of environments down. Be aware of this if you like to tinker with the code of your dependencies - your changes may affect other environments!
+
+The software in an environment can be automatically upgraded:
+
+```shell-session
+$ mamba upgrade -n <name> --all
+```
+
+But that can go wrong if you're mixing channels. Recreating the environment often works better:
+
+```shell-session
+$ mamba env remove -n <name>
+$ mamba create -n <name> -c conda-forge <package(s)>
+```
+
+Or specifying the particular package you want to update:
+
+```shell-session
+$ mamba upgrade -n <name> <package>
+```
+
+### Combining channels
+
+Sometimes, some of the software you want to use is not available on Conda Forge. In these cases, you can either install it in the usual way for your OS, or you can try combining channels:
+
+```shell-session
+$ mamba create -n psi4_and_bespokefit -c psi4 -c conda-forge -c anaconda openff-bespokefit psi4
+```
+
+In this case, this doesn't work because Conda/Mamba has to be very clever about how it solves this environment. It has to be able to choose versions of dependencies from the `anaconda` channel, even when those dependencies exist in the higher-priority `psi4` channel. It's generally recommended to use [strict channel priority] to avoid conflicts, but sometimes an environment can only be solved when channel priority is set to `"flexible"`. The catch is that the environment may solve and install but not work! But it's usually worth a try.
```

Comment:
> The catch is that the environment may solve and install but not work!

Is this true? I'm not sure I've ever had this happen to me. Usually my issues with mixed channels relate to the environment not solving at all.

Reply:
I'm sure I had a problem last week where a BespokeFit+Psi4 environment solved with flexible priority, but then I got missing symbol errors when trying to run it. And I believe this is one of the motivations behind strict. But it's working today so maybe I'm misremembering.

Reply:
Hi There. FYI I had a day off yesterday, so only just starting to look at this. I will send my comments to Josh by email.